admin/aioseop_module_class.php

```diff
@@ -1829,13 +1791,28 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 				wp_enqueue_media();
 			}
 
+			$helper_dep = array(
+				'jquery',
+				'jquery-ui-core',
+				'jquery-ui-widget',
```

Comment:
is widget required?

Reply:
It is required, if you look under Dependencies. http://api.jqueryui.com/tooltip/